Resources/Prototypes/Entities/Mobs/NPCs/animals.yml

```diff
@@ -2592,6 +2592,111 @@
     attributes:
       gender: epicene
 
+- type: entity
+  name: Muhtar
+  parent: SimpleMobBase
+  id: MobMuhtar
+  description: "Man's best friend. And the worst enemy of the enemy of NanoTraisen."
+  components:
+  - type: Sprite
+    drawdepth: Mobs
+    sprite: Mobs/Pets/muhtar.rsi
+    layers:
+    - map: ["enum.DamageStateVisualLayers.Base"]
+      state: muhtar
+    - type: Physics
+    fixtures:
+      fix1:
+        shape:
+          !type:PhysShapeCircle
+          radius: 0.35
+        density: 20
+        mask:
+        - MobMask
+        layer:
+        - MobLayer
+  - type: Inventory
+    speciesId: sheepdog
+    templateId: pet
+  - type: InventorySlots
+  - type: Strippable
+  - type: UserInterface
+    interfaces:
+    - key: enum.StrippingUiKey.Key
+      type: StrippableBoundUserInterface
+  - type: DamageStateVisuals
+    states:
+      Alive:
+        Base: muhtar
+      Critical:
+        Base: muhtar_dead
+      Dead:
+        Base: muhtar_dead
+  - type: Butcherable
+    spawned:
+    - id: FoodMeat
+      amount: 2
+  - type: ReplacementAccent
+    accent: dog
+  - type: InteractionPopup
+    successChance: 1
+    interactSuccessString: petting-success-dog # It mustn't react at it at all.
+    interactFailureString: petting-failure-generic
+  - type: Grammar
+    attributes:
+      proper: true
+      gender: male
+  - type: Tag
+    tags:
+    - CannotSuicide
+    - VimPilot
+    - FootstepSound
+    - DoorBumpOpener
+  - type: GhostTakeoverAvailable
+  - type: GhostRole
+    makeSentient: true
+    allowMovement: true
+    requirements:
+      - !type:DepartmentTimeRequirement
+        department: Security
+        time: 36000
+    name: ghost-role-information-muhtar-name
+    description: ghost-role-information-muhtar-descriprion
+    rules: ghost-role-information-muhtar-rules
+  - type: MeleeWeapon
+    altDisarm: false
```

Comment:
Why false? Disarming seems pretty in-line for a police dog.

Reply:
> Why false? Disarming seems pretty in-line for a police dog.

Playtesters' told me that it was hard to hit anybody with LMB. Moreover it has 30 stamina damage, it wont be that hard to stamcrit anybody.

Reply:
And I actualy dont see a way of any dog disarming anybody without harming them. Is it even possible when human is resisting?